server/webhook.go

```diff
@@ -199,3 +207,44 @@ func (p *Plugin) GetWebhookURL(jiraURL string, teamID, channelID string) (subURL
 
 	return subURL, legacyURL, nil
 }
+
+func (wh *webhook) applyReporterNotification(bb []byte, reporter *jira.User) error {
+	if wh.eventTypes.ContainsAny("event_created_comment", "event_updated_comment") {
```

Comment:
To keep the indentation of this function flat:

```suggestion
	if !wh.eventTypes.ContainsAny("event_created_comment") {
        return
    }
```